src/sendGCM.js

```diff
@@ -91,7 +91,8 @@ const sendGCM = (regIds, data, settings) => {
     body_loc_args: data.locArgs, // Android, iOS
     title_loc_key: data.titleLocKey, // Android, iOS
     title_loc_args: data.titleLocArgs, // Android, iOS
-    android_channel_id: data.android_channel_id, // Android
+    channel_id: data.android_channel_id, // Android
+    notification_count: data.notificationCount || data.badge, // Android
```

Comment:
do you use `badge` here because it makes easier to use if both iOS and Android need to be supported?